index.html

```diff
@@ -9,16 +9,18 @@
   </head>
   <body>
     <div id="left">
-      <table id="navigation">
-        <tr>
-          <td colspan="3"><a id="toc">Table of Contents</a></td>
-        </tr>
-        <tr>
-          <td><a id="previous">&lt; Previous</a></td>
-          <td><a id="up">^ Up</a></td>
-          <td><a id="next">Next &gt;</a></td>
-        </tr>
-      </table>
+      <div id="navigation">
+        <table>
+          <tr>
+            <td colspan="3"><a id="toc" title="Table of Contents">&#x25B3;</a></td>
```

Comment:
What about using this symbol instead: <b>&equiv;</b> `<b>&equiv;</b>` for the table of contents?